src/testData/mockData.json

```diff
@@ -0,0 +1,101 @@
+{
+    "DEFAULT_URL": "https://unstoppabledomains.com/api/v1",
```

Comment:
I think that should not be part of the mock file: there is no reason to add a complexity inside test.